src/components/AnchorNav/index.css

```
@@ -82,6 +82,10 @@
   }
 }
 
+.side-nav#anchor-nav-large-font {
```

can the 2 blocks be merged into a single one? 

ie merge this with line 107?

```suggestion
.side-nav#anchor-nav-large-font {
  margin-bottom: var(--ic-space-md);
  width: 100%;
}
```

---

Fixed, missed this when I re-factored the code